crawl-ref/source/dat/strings/de/ui.txt

```diff
@@ -37,7 +37,7 @@ Choose Game Seed
 Spiel-Seed wählen
 %%%%
 Play with a chosen custom dungeon seed.
-Spielen mit einer gewählten Seed für Erzeugung des Dungeons
+Spielen mit einer gewählten Seed für Erzeugung des Kerkers
```

Comment:
Spiele mit einer gewählten Saat für Erzeugung des Kerkers

Reply:
Will do in separate PR